skyvern/forge/agent.py

```
@@ -160,6 +161,65 @@ class ForgeAgent:
+                # Skip the current run
+                if run.workflow_run_id == current_workflow_run_id:
+                    continue
+
+                try:
+                    # Get downloaded files for this run
+                    downloaded_files = await app.STORAGE.get_downloaded_files(
+                        organization_id=organization_id,
+                        run_id=run.workflow_run_id,
+                    )
+                    all_downloaded_files.extend(downloaded_files)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)